Gemfile

```diff
@@ -4,7 +4,9 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'activefedora-aggregation', github: 'projecthydra-labs/activefedora-aggregation'
-gem 'active-fedora', github: 'projecthydra/active_fedora', branch: 'master'
+gem 'activefedora-aggregation', github: 'projecthydra-labs/activefedora-aggregation'
```

Comment:
This is a duplicate of line 6. Could you strike line 6 and specify a `ref` here?